### PR TITLE
Add Archery and its patches

### PR DESCRIPTION
**Pull request by sean-kang**
This PR is to add Archery mod and its patches to the master list. Thanks.

Archery: https://www.nexusmods.com/skyrimspecialedition/mods/10926/

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -4862,6 +4862,46 @@ plugins:
...
+      - crc: 0xB2AD660F
+        util: 'SSEEdit v4.0.2'
+      # version: 2.4
+      - crc: 0x0DDC51B6
+        util: 'SSEEdit v4.0.2'
+  - name: 'Archery_Bruma_Patch.esp'
+    req:
+      - 'ArcheryV2.esp'
+    after:
+      - 'BSHeartland.esm'
```
Comment (MacSplody) [inline masterlist.yaml]: An `after` for `ESM` or plugin masters in not required.

File: masterlist.yaml [hunk 2]
```diff
@@ -4862,6 +4862,46 @@ plugins:
...
+      - crc: 0x0470CE8E
+        util: 'SSEEdit v4.0.2'
+      # version: 1.1 Aleyid Arrows
+      - crc: 0xEF057475
+        util: 'SSEEdit v4.0.2'
+  - name: 'Archery_Campfire_Patch.esp'
+    req:
+      - 'ArcheryV2.esp'
+    after:
+      - 'Campfire.esm'
```
Comment (MacSplody) [inline masterlist.yaml]: Same as above.

**Review by MacSplody — CHANGES_REQUESTED** [review]
The rest looks good, just the two afters.

File: masterlist.yaml (hunk 1 above)
Comment (sean-kang) [reply masterlist.yaml]: Removed `after`.

File: masterlist.yaml (hunk 2 above)
Comment (sean-kang) [reply masterlist.yaml]: Removed `after`.

**Review by MacSplody — APPROVED** [review]

**Comment by MacSplody** [comment]
@sean-kang Thanks

**Pull request merged**